doc/nrf/app_dev/device_guides/nrf70/nrf7002eb2_dev_guide.rst

```diff
@@ -50,40 +50,32 @@ The board is mounted on the **P1** connector of these DKs
       +-----------------------------------+-------------------+-----------------------------------------------+
-      | MISO (D1)                         | P1.05             | SPI MISO                                      |
+      | MISO (D1)                         | P3.01             | SPI MISO                                      |
       +-----------------------------------+-------------------+-----------------------------------------------+
-      | BUCKEN (EN)                       | P1.02             | Enable Buck regulator                         |
+      | BUCKEN (EN)                       | P1.04             | Enable Buck regulator                         |
       +-----------------------------------+-------------------+-----------------------------------------------+
-      | IOVDDEN                           | P1.04             | Enable IOVDD regulator                        |
+      | IOVDDEN                           | P1.13             | Enable IOVDD regulator                        |
       +-----------------------------------+-------------------+-----------------------------------------------+
-      | IRQ (IRQ)                         | P1.10             | Interrupt from nRF7002                        |
+      | IRQ (IRQ)                         | P1.05             | Interrupt from nRF7002                        |
       +-----------------------------------+-------------------+-----------------------------------------------+
-      | GRANT (GRT)                       | P1.03             | Coexistence grant from nRF7002                |
+      | GRANT (GRT)                       | P1.07             | Coexistence grant from nRF7002                |
       +-----------------------------------+-------------------+-----------------------------------------------+
-      | REQ (REQ)                         | P1.09             | Coexistence request to nRF7002                |
+      | REQ (REQ)                         | P0.04             | Coexistence request to nRF7002                |
       +-----------------------------------+-------------------+-----------------------------------------------+
-      | STATUS (ST0)                      | P1.08             | Coexistence status from nRF7002               |
+      | STATUS (ST0)                      | P0.03             | Coexistence status from nRF7002               |
       +-----------------------------------+-------------------+-----------------------------------------------+
 
-DIP switch settings for nRF54L15 DK and nRF54H20 DK
-***************************************************
-
-The **GPIO MAP SEL** switch on the nRF7002-EB II can be used to select the GPIO mapping for the nRF54H20 DK and the nRF54L15 DK.
-
-* **nRF54L15 DK**: Set the switch towards MAP1 (the LED will glow at the **MAP1** end).
-* **nRF54H20 DK**: Set the switch towards MAP2 (the LED will glow at the **MAP2** end).
-
 .. _nrf7002eb2_building_programming:
 
 Building and programming with nRF7002-EB II
```

Comment:
I am suggesting to use a little bit better language on the below text. Strictly, speaking we do not build for a shield target , we always build for a SoC. I left a comment below, but i realize there's one more place we can improve the language.

Suggestion here

To build an application with support for Wi-Fi using the nRF7002-EB II, use a compatible :ref:`board target <app_boards_names>` with the CMake ``SHIELD`` option set to the corresponding shield name.
See :ref:`cmake_options` for instructions on how to provide CMake options.